llvm/README.md

```diff
@@ -73,11 +75,8 @@ public class Factorial {
 
     public static void main(String[] args) {
         // Stage 1: Initialize LLVM components
-        LLVMInitializeCore(LLVMGetGlobalPassRegistry());
-        LLVMLinkInMCJIT();
-        LLVMInitializeNativeAsmPrinter();
-        LLVMInitializeNativeAsmParser();
-        LLVMInitializeNativeTarget();
+	    LLVMInitializeNativeTarget();
+	    LLVMInitializeNativeAsmPrinter();
```

Comment:
Please fix other formatting issues here and there such as here while you're at it